cmake/CMakeLists.txt

```diff
@@ -151,3 +158,11 @@ if (protobuf_BUILD_TESTS)
 endif (protobuf_BUILD_TESTS)
 
 include(install.cmake)
+
+if (protobuf_BUILD_EXAMPLES)
+  include(examples.cmake)
+endif (protobuf_BUILD_EXAMPLES)
```

Comment:
I do not like it, but this style was borrowed from the main list, which was created before me.
I would like to leave it now and work on functionality, and then make corrections of style in all scripts separate PR.